src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -72,6 +78,24 @@ export class RundownComponent implements OnInit, OnDestroy, OnChanges {
       .catch(error => this.logger.data(error).error('Failed subscribing to rundown actions changes.'))
   }
 
+  public scrollViewToSegment(event: PartEvent): void {
+    const element: HTMLElement | null = document.getElementById(event.segmentId)
+    if (!element || this.isInsideViewport(element)) {
+      return
+    }
+
+    document.getElementById(event.segmentId)?.scrollIntoView({
+      behavior: 'smooth',
+      block: 'nearest',
+      inline: 'nearest',
```

Comment:
These are "almost" magic strings.
It would be nice if we could extract this block to something like:
```
const SCROLL_CONFIGURATION: ScrollIntoViewOptions = {
  behavior: 'smooth',
  block: 'nearest',
  inline: 'nearest',
}
```
so you can use it like:
`element.scrollIntoView(SCROLL_CONFIGURATION)`

Reply:
It has been moved to a constant, i called it `SMOOTH_SCROLL_CONFIGURATION` to make it specific as to what the configuration is specifically for.